src/AElf.Runtime.WebAssembly/Extensions/Extensions.cs

```diff
@@ -0,0 +1,24 @@
+namespace AElf.Runtime.WebAssembly.Extensions;
+
+public static class Extensions
+{
+    public static bool Contains(this CallFlags callFlags, CallFlags other)
+    {
+        return (callFlags & other) == other;
+    }
+
+    /// <summary>
+    /// TODO: Improve
```

Comment:
just to check. is this TODO for a future PR?

Reply:
Yeah currently I don't know how other ReturnCode will be returned